core/src/main/java/io/aiven/klaw/helpers/db/rdbms/InsertDataJdbc.java

```diff
@@ -185,7 +185,10 @@ public synchronized String insertIntoTopicSOT(List<Topic> topics) {
       topicRepo.save(topic);
     }
 
-    return ApiResultStatus.SUCCESS.value;
+    return CRUDResponse.<Topic>builder()
+        .resultStatus(ApiResultStatus.SUCCESS.value)
+        .entities(topics)
```

Comment:
does it also consider existingTopicIds ? defined in line 166

Reply:
So the existingTopicIds is a a subset of topics, so all the changes should be in there however I will create a unit test to verify that.